applications/SwimmingDEMApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -126,8 +114,39 @@ bool ModelPartHasNodalVariableOrNot(VariableChecker& rChecker, ModelPart& rModel
 
 namespace py = pybind11;
 
+// template<std::size_t TDim, class TParticle>
+// void DeclareBinbasedCoupler(py::module &m, std::string sphere_type)
+// {
+// 	using Class = BinBasedDEMFluidCoupledMapping<TDim,TParticle>;
+// 	std::string pyclass_name = std::string("BinBased")
+// 							 + sphere_type
+// 							 + std::string("FluidCoupledMapping")
+// 							 + std::to_string(TDim)
+// 							 + std::string("D");
+// 	py::class_<Class>(m, pyclass_name.c_str(), py::buffer_protocol(), py::dynamic_attr())
+// 	.def(py::init<Parameters&>())
+// 	.def("InterpolateVelocityOnAuxVelocity", &Class::InterpolateVelocityOnAuxVelocity)
+// 	.def("ImposeVelocityOnDEMFromFieldToAuxVelocity", &Class::ImposeVelocityOnDEMFromFieldToAuxVelocity)
+// 	.def("InterpolateFromFluidMesh", &Class::InterpolateFromFluidMesh)
+// 	.def("ImposeFlowOnDEMFromField", &Class::ImposeFlowOnDEMFromField)
+// 	.def("InterpolateFromDEMMesh", &Class::InterpolateFromDEMMesh)
+// 	.def("HomogenizeFromDEMMesh", &Class::HomogenizeFromDEMMesh)
+// 	.def("ComputePostProcessResults", &Class::ComputePostProcessResults)
+// 	.def("AddDEMCouplingVariable", &BinBasedDEMFluidCoupledMapping<3,SphericParticle>::AddDEMCouplingVariable<double>)
+// 	.def("AddDEMCouplingVariable", &BinBasedDEMFluidCoupledMapping <3,SphericParticle>::AddDEMCouplingVariable<array_1d<double, 3>>)
+// 	.def("AddFluidCouplingVariable", &BinBasedDEMFluidCoupledMapping <3,SphericParticle>::AddFluidCouplingVariable<double>)
+// 	.def("AddFluidCouplingVariable", &BinBasedDEMFluidCoupledMapping <3,SphericParticle>::AddFluidCouplingVariable<array_1d<double, 3>>)
+// 	.def("AddDEMVariablesToImpose", &BinBasedDEMFluidCoupledMapping <3,SphericParticle>::AddDEMVariablesToImpose<double>)
+// 	.def("AddDEMVariablesToImpose", &BinBasedDEMFluidCoupledMapping <3,SphericParticle>::AddDEMVariablesToImpose<array_1d<double, 3>>)
+// 	.def("AddFluidVariableToBeTimeFiltered", &BinBasedDEMFluidCoupledMapping <3,SphericParticle>::AddFluidVariableToBeTimeFiltered<double>)
+// 	.def("AddFluidVariableToBeTimeFiltered", &BinBasedDEMFluidCoupledMapping <3,SphericParticle>::AddFluidVariableToBeTimeFiltered<array_1d<double, 3>>);
+// }
+
```

Comment:
Remove commented lines, please

Reply:
I wanted to keep them since this could be fixed...